app/src/main/java/com/pluu/webtoon/data/daum/DaumDetailApi.kt

```diff
@@ -0,0 +1,100 @@
+package com.pluu.webtoon.data.daum
+
+import com.pluu.kotlin.iterator
+import com.pluu.webtoon.data.DetailRequest
+import com.pluu.webtoon.data.IRequest
+import com.pluu.webtoon.data.REQUEST_METHOD
+import com.pluu.webtoon.data.impl.AbstractDetailApi
+import com.pluu.webtoon.di.NetworkUseCase
+import com.pluu.webtoon.item.*
+import com.pluu.webtoon.utils.safeAPi
+import org.json.JSONObject
+
+/**
+ * 다음 웹툰 상세 API
+ * Created by pluu on 2017-04-22.
+ */
+class DaumDetailApi(
+    networkUseCase: NetworkUseCase
+) : AbstractDetailApi(networkUseCase) {
+    override fun parseDetail(param: DetailRequest): DetailResult {
+        val id = param.episodeId
+
+        ///////////////////////////////////////////////////////////////////////////
+        // API
+        ///////////////////////////////////////////////////////////////////////////
+
+        val apiResult = safeAPi(requestApi(createApi(id))) { response ->
+            JSONObject(response)
+        }
+
+        val responseData = when (apiResult) {
+            is Result.Success -> apiResult.data
+            is Result.Error -> return DetailResult.ErrorResult(ERROR_TYPE.DEFAULT_ERROR)
+        }
+
+        ///////////////////////////////////////////////////////////////////////////
+        // Parse Data
+        ///////////////////////////////////////////////////////////////////////////
+
+        val data =
+            responseData.optJSONObject("data") ?: return DetailResult.ErrorResult(ERROR_TYPE.NOT_SUPPORT)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Pluu/WebToon/blob/e6d7b5dbb11d2d4fb917c63fbb0187d6361e5aea/app/src/main/java/com/pluu/webtoon/data/daum/DaumDetailApi.kt#L41"></span>Exceeded max line length (100) (cannot be auto-corrected)</td>
    </tr>
  </tbody>
</table>